pkgs/development/tools/documentation/doxygen/default.nix

```
@@ -19,16 +30,24 @@ stdenv.mkDerivation rec {
   ];
 
   buildInputs = [ libiconv ]
+    ++ lib.optionals withSqlite [ sqlite.dev ]
```

```suggestion
    ++ lib.optionals withSqlite [ sqlite ]
```
this should work, too

---

Adopted change locally; will be fixed next time I force-push.